lib/livebook_web/live/sidebar_helpers.ex

```diff
@@ -101,6 +110,56 @@ defmodule LivebookWeb.SidebarHelpers do
     """
   end
 
+  defp hub_section(assigns) do
+    ~H"""
+    <%= if Application.get_env(:livebook, :feature_flags)[:hub] do %>
+      <div id="sidebar--hub" class="flex flex-col mt-12">
```

Comment:
```suggestion
      <div id="hubs" class="flex flex-col mt-12">
```

or `sidebar-hubs` :)